data/bboxes.js

```diff
@@ -0,0 +1,2954 @@
+var bboxes = {
```

Comment:
That's a @meganrm q I believe. 

I'm all for throwing this along with the other files In `data/` into the database itself if that works too.